Sources/CAudioKit/Sequencing/SequencerEngine.mm

```diff
@@ -9,10 +9,16 @@
 #include <stdio.h>
 #include <atomic>
 #include <mutex>
+#include "TPCircularBuffer.h"
 
 #define NOTEON 0x90
 #define NOTEOFF 0x80
 
+struct SequencerEvent {
+    bool notesOff = false;
+    double seekPosition = NAN;
```

Comment:
thats cool i've never seen NAN used on purpose

Reply:
First encountered NAN as a sentinel in this book: https://www.oreilly.com/library/view/21st-century-c/9781491904428/